src/dockerclient.cc

```diff
@@ -0,0 +1,42 @@
+#include "dockerclient.h"
+#include <logging/logging.h>
+#include "http/httpclient.h"
+#include "utilities/utils.h"
+
+namespace Docker {
+
+DockerClient::DockerClient(const std::string& app, bool curl) {
+  if (curl) {
+    std::string cmd = "/usr/bin/curl --unix-socket /var/run/docker.sock http://localhost/containers/json?" + app;
```

Comment:
I guess it can be removed now?

Reply:
why not leave it as an option? it also serves as documentation IMO